cmake/scripts/expand_instantiations.cc

```diff
@@ -386,9 +386,9 @@ void substitute (const std::string &text,
           // between 0 and SPLIT_INSTANTIATIONS_COUNT-1 to get the instantiations of that
           // particular chunk.
           static unsigned int counter = 0;
-          std::cout << "#if !defined(SPLIT_INSTANTIATIONS_COUNT) || ("
+          std::cout << "#if (("
                     << counter++
-                    << " % SPLIT_INSTANTIATIONS_COUNT == SPLIT_INSTANTIATIONS_INDEX)" << std::endl;
+                    << " % SPLIT_INSTANTIATIONS_COUNT) == SPLIT_INSTANTIATIONS_INDEX)" << std::endl;
```

Comment:
I know, that was not supposed to fix it (you skipped the other part of the patch). Anyways, that also didn't work...